types/react-permissible.d.ts

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+
+declare module '@brainhubeu/react-permissible' {
+
```

Comment:
How about instead of arguing about style, I just add https://prettier.io/ and apply it incrementally to all changed files and we'll call it a day? :thinking: 

Alternative option: add prettier to the entire project, in a separate PR.

Reply:
I'd use our already configured eslint if it's possible. So that, we won't finish with the repo that half of the files are using prettier and half not. Second thing is that it's additional dependency, that we would like to avoid (I know that's just devDependency, but still something new to configure and maintain).

So, if it's possible, please follow our eslint rules, which should already report that empty line as an error.